x/evmutil/types/msg.go

```diff
@@ -123,7 +123,7 @@ func (msg MsgConvertERC20ToCoin) ValidateBasic() error {
 		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, "receiver is not a valid bech32 address")
 	}
 
-	if msg.Amount.LTE(sdk.ZeroInt()) {
+	if !msg.Amount.IsNil() && msg.Amount.LTE(sdk.ZeroInt()) {
```

Comment:
```suggestion
	if msg.Amount.IsNil() || msg.Amount.LTE(sdk.ZeroInt()) {
```
looks like this lets a nil value through. I think an OR exits early if it finds a `true` so should never call `.LTE(...)` on a nil value.